grpc/src/main/java/com/linecorp/armeria/server/grpc/FramedGrpcService.java

```diff
@@ -217,20 +217,14 @@ protected HttpResponse doPost(ServiceRequestContext ctx, HttpRequest req) throws
 
         ServerMethodDefinition<?, ?> method = lookupMethodFromAttribute ? ctx.attr(RESOLVED_GRPC_METHOD) : null;
         if (method == null) {
-            final String methodName = GrpcRequestUtil.determineMethod(ctx);
-            if (methodName == null) {
-                return HttpResponse.of(HttpStatus.BAD_REQUEST,
```

Comment:
Let's say that we bind a method to "/prefix/package/Method".
Two routes can be created for the method:
- `Route.builder().exact("/prefix/package/Method).build()`
- `Route.builder().pathePrefix("/prefix).build()`

If a request is sent to `/prefix/another/path`, the second route would be used and this is the case that we return 404. We used to return 400 but I thought 404 is more rational because the `/prefix/another/path` path is not found.

Reply:
Ah yes, my concern was more about returning 400 when the path isn't prepended with a slash (which was the previous behavior)

I double-checked, and it seems we also validate this at the decoder level, so this change looks good to me 👍